workers/unity/Assets/Generated/Improbable.Gdk.Generated.asmdef

```diff
@@ -1,6 +1,13 @@
 {
     "name": "Improbable.Gdk.Generated",
-    "references": [ "Unity.Collections", "Unity.Entities", "Unity.Mathematics", "Unity.Entities.Hybrid", "Improbable.Gdk.Core" ],
+    "references": [ 
+        "Unity.Collections", 
+        "Unity.Entities", 
+        "Unity.Mathematics", 
+        "Unity.Entities.Hybrid",
+        "Improbable.Gdk.Core",
+        "Improbable.Gdk.Tools"
```

Comment:
Do we need this here since the build stuff is in its own assembly?

Reply:
I can revert it. it's just a formatting change

Reply:
Oh no - the `Improbable.Gdk.Tools` reference

Reply:
Or was that required before.. ?

Reply:
oh, good point. not sure what happened there, will check